pkg/domain/stateDiffGlobalConfig.go

```diff
@@ -0,0 +1,42 @@
+package domain
```

Comment:
🚫 **[checkstyle]** <dupl> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>1-42 lines are duplicate of `pkg/domain/stateDiffDoguConfig.go:1-41`